std-bits/table/src/main/java/org/enso/table/data/column/operation/map/UnaryMapOperation.java

```diff
@@ -3,24 +3,21 @@
 import org.enso.table.data.column.storage.Storage;
 
 /**
- * A map-like operation that ignores its second argument
+ * A unary map-like operation.
  *
  * @param <I> the supported storage type
  */
-public abstract class UnaryMapOperation<T, I extends Storage<T>> extends MapOperation<T, I> {
+public abstract class UnaryMapOperation<T, I extends Storage<? super T>> {
+  private final String name;
+
   public UnaryMapOperation(String name) {
-    super(name);
+    this.name = name;
   }
 
-  protected abstract Storage<?> run(I storage);
-
-  @Override
-  public Storage<?> runMap(I storage, Object arg, MapOperationProblemBuilder problemBuilder) {
-    return run(storage);
-  }
+  protected abstract Storage<?> run(I storage, MapOperationProblemBuilder problemBuilder);
```

Comment:
Lets either call this `runUnaryMap` to be consistent with `BinaryMapOperation` or better I think to rename `runBiMap` to just be `run` as well.

Reply:
But `runBiMap` has 2 variants - `runBiMap` and `runZip`, if we rename one of these to just `run` IMO it will be less clear what is the relation between `run` and `runZip`. So I'd go for `runUnaryMap` here.